pkg/interceptors/cel/cel_test.go

```diff
@@ -156,13 +186,14 @@ func TestInterceptor_ExecuteTrigger(t *testing.T) {
 	}
 }
 
-func TestFilterEvaluation(t *testing.T) {
+func TestExpressionEvaluation(t *testing.T) {
+	testSHA := "ec26c3e57ca3a959ca5aad62de7213c562f8c821"
 	jsonMap := map[string]interface{}{
 		"value": "testing",
-		"sha":   "ec26c3e57ca3a959ca5aad62de7213c562f8c821",
+		"sha":   testSHA,
 	}
 	header := http.Header{}
-	evalEnv := map[string]interface{}{"body": jsonMap, "header": header}
+	evalEnv := map[string]interface{}{"body": jsonMap, "headers": header}
```

Comment:
Is it `header` or `headers`? In TriggerBindings we use `header` I think we should be consistent!